packages/tldraw/src/hooks/useStylesheet.ts

```diff
@@ -2,18 +2,16 @@ import * as React from 'react'
 
 const styles = new Map<string, HTMLStyleElement>()
 
-const UID = `Tldraw-fonts`
+const UID = `tldraw-fonts`
 const WEBFONT_URL =
```

Comment:
If this can be Array & we should allow loading multiple font that is better.

Reply:
I'd like to have it all come in via CSS alone, but unfortunately we need to know a lot about font styles (in JavaScript) in order to make text measurement work. Custom fonts would need to be a separate issue but worth investigating.